ZAPD/HighLevel/HLModelIntermediette.cpp

```diff
@@ -21,6 +21,7 @@ HLModelIntermediette::HLModelIntermediette()
 	startIndex = 0;
 	meshStartIndex = 0;
 	hasSkeleton = false;
+	lastTrans = Vec3s(0, 0, 0);
 }
```

Comment:
Trivials contructors like this one can be replaced with
`HLModelIntermediette() = default;` 
in the header.